src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionState.java

```diff
@@ -61,25 +61,11 @@ public double segmentMinorFraction(final int segment) {
         return get(AlleleFractionParameter.MINOR_ALLELE_FRACTIONS, MinorFractions.class).get(segment);
     }
 
-    //copy the state with reference to the SAME minorFractions list data (to save copying) but different value of
-    //one of the scalar parameters
-    //This is dangerous and minorFractions should not be modified in the copy.
-    //
-    //The purpose of this is to make an MCMC proposal state for calculating a likelihood with one of the scalar parameters
-    //modified (these are unboxed in the getters, so changing these in the copy is safe)
-    protected AlleleFractionState shallowCopyWithProposedMeanBias(final double proposedMeanBias) {
-        return new AlleleFractionState(proposedMeanBias, biasVariance(), outlierProbability(), minorFractions());
+    public AllelicBiasParameters getParameters() {
```

Comment:
Again, perhaps `getGlobalParameters` or something more specific, since the minor fractions are also referred to as parameters in many places.